massa-api/src/public.rs

```diff
@@ -138,7 +138,7 @@ impl MassaRpcServer for API<Public> {
         {
             let address = address.unwrap_or_else(|| {
                 // if no addr provided, use a random one
-                Address::from_public_key(&KeyPair::generate().get_public_key())
+                Address::from_public_key(&KeyPair::generate(0).unwrap().get_public_key())
```

Comment:
Leave comment/use `expect` to provide context for the next person coming around to remove the unwraps

Reply:
I'll try and see what I can do better here, thanks!